### ISSUE TITLE
Items Not Loading When Small itemsPerPage

### ISSUE THREAD

**Issue by isyedaliraza** [marciovalim/realtime_pagination#6]
Hi, I am using this plugin in one of my projects. I have noticed an issue that until the items don't overflow the page the infinite loading will not work. Let's say we have `itemsPerPage: 5` and these items take half of the screen's total height then more items are not loaded. When we set `itemsPerPage: 10` and the list overflows then more items are loaded.

**Comment by marciovalim** [comment]
Hi. Yes, this is the behavior of pagination. You only load a single page of content and then another.

**Comment by marciovalim** [comment]
Hi @isyedaliraza. Yes, this is the behavior of pagination. You only load a single page of content and then another.

**Comment by isyedaliraza** [comment]
@marciovalim I think I didn't explain the problem well. If the list view doesn't overflow for the first time, then the load more items callback is not triggered. To reproduce the issue just set the `itemsPerPage` to a smaller value e.g. 3 or 4 etc. I hope this issue is clear now.

**Comment by marciovalim** [comment]
@isyedaliraza I understand that the itemsPerPage should be a number that makes the page overflow the items. I just don't understand why this is a problem for you. Just increase itemsPerPage and it's solved.
I don't think this is a issue, because the purpose of pagination is to load one page at time. So if it's needed 7 items to fill the page, set itemsPerPage as 7. If it's needed only 4, set as 4.

**Comment by isyedaliraza** [comment]
@marciovalim I agree that setting `itemPerPage` to a higher number fixes the issue. But I faced this issue when I kept `itemsPerPage` to 10. On smaller screen phones 10 items can't fit the screen so the overflow was causing more items to load but for big-screen phones, 10 items can't make the screen overflow so no more items will load only the initial 10 items will be loaded. I will try to investigate if we can cover this edge case and submit a PR.
